server/src/main/java/org/candlepin/dto/api/v1/ContentOverrideDTO.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) 2009 - 2018 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.TimestampedCandlepinDTO;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+
+
+/**
+ * The ContentOverrideDTO is used as a DTO for the ContentOverride class and the basis for its
+ * subclasses.
+ *
+ * @param <T>
+ *  The type of this content override
+ */
+public class ContentOverrideDTO<T extends ContentOverrideDTO> extends TimestampedCandlepinDTO<T> {
+
+    public static final long serialVersionUID = 1L;
+
+    protected String contentLabel;
+    protected String name;
+    protected String value;
+
+    /**
+     * Initializes a new ContentOverrideDTO instance with null values.
+     */
+    public ContentOverrideDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new ContentOverrideDTO instance which is a shallow copy of the provided
+     * source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public ContentOverrideDTO(T source) {
+        this.populate(source);
```

Comment:
Same as the subclasses, this should be `super(source)`

Reply:
Fixed.